internal/broker/plans.go

```diff
@@ -780,6 +780,10 @@ func IsOwnClusterPlan(planID string) bool {
 	return planID == OwnClusterPlanID
 }
 
+func IsShared(planId string) bool {
```

Comment:
How will we deal with shared plan concept having "being shared" property configurable and with granularity different than plan?